WalletWasabi.Documentation/Guides/Wasabi2TestVectors.md

```diff
@@ -0,0 +1,114 @@
+# Wasabi Wallet 2.0 Test Vectors
+
+Thank you for considering to test and review Wasabi Wallet 2.0, it's been a long time in the making and we appreciate every feedback so close before release.
+
+Many of the tests can be done on testnet, but some require a regtest environment. Follow the setup instructions [here](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Documentation/WasabiSetupRegtest.md).
+
+## Installation
+
+- Download the release candidate package, public key and signature
+- Verify signature
+- Install the package
+
```

Comment:
Maybe elaborate how to verify signature?

Reply:
Since it depends on each platform, probably better to just add a link to the docs...

```suggestion
## Installation

- Download the release candidate package, public key and signature
- Verify signature [explained [here](https://docs.wasabiwallet.io/using-wasabi/InstallPackage.html)]
- Install the package

```